docs/content/staking/custody-providers.mdx

```diff
@@ -0,0 +1,92 @@
+on Flow.
+
+## Blocto
+
+For details about how to use your Blocto account to stake and earn rewards you'll
+find everything you need to know in Blocto's [staking guide](https://guide.blocto.app/article/stake-flow-tokens-step-by-step-with-blocto).
+
+#### Auto-restaking: Optional
+
+With Blocto, users have the option to re-stake earned rewards, or not.
+
+#### Delegating
+
+Blocto allows users to delegate their locked FLOW as stake to a Flow node of their choice by providing the node's unique ID. This can be done using the Flow Port.
+For details about how to delegate using Flow Port, please see our [step-by-step](/) guide.
+
+#### Withdrawing Rewards
+
+Users using the Blocto wallet can also withdraw their rewards using the Flow Port!
+
+---
+
+## Ledger
+
+If you're storing your locked FLOW tokens using a Ledger device, you can use the Flow Port to **stake** or **delegate** them to earn rewards.
+For more details you can read [this official post](https://www.onflow.org/post/flow-port-is-here) to familiarize yourself with the Flow Port app,
+and how to use it as a staking portal.
+
+#### Auto-restaking: No
+
+Flow Port does not support auto-restaking of your rewards. Follow the Flow Port [staking guide](/) for instructions about how to stake using the Port.
+
+#### Delegating
+
+Follow the Flow Port [staking guide](/) for instructions about how to delegate stake using the Port.
+
+#### Withdrawing Rewards
+
+Use [Flow Port](https://port.onflow.org) to withdraw any rewards you earned by staking or delegating locked FLOW form your Ledger device.
+
+---
+
+## Coinlist
+
+Coinlist is operating FLOW nodes and will automatically delegate your locked FLOW to them as stake.
+If you would like to withdraw your rewards, you'll need to contact Coinlist's support team.
+
+#### Auto-restaking: Yes
+
+Coinlist will automatically re-stake any rewards you earn. If you'd like to withdraw or delegate stake to another node, you'll need to
+contact Coinlist's support team by email.
+
+#### Delegating
+
+If you would like to delegate our locked FLOW to another node of your choosing, you'll need to contact Coinlist's support team.
+
+#### Withdrawing Rewards
+
+If you would like to withdraw your rewards, you'll need to contact Coinlist's support team.
+
+---
+
+## Kraken
+
+Kraken is operating FLOW nodes and will automatically delegate your locked FLOW to them as stake.
+For details about how to delegate, or withdraw your rewards, you'll need to contact Kraken's support team by email.
+
+#### Auto-restaking: Yes
+
+Kraken will automatically re-stake any rewards you earn.
+
+#### Delegating
+
+If you would like to delegate your locked FLOW to another node of your choosing, you'll need to contact Kraken's support team.
+
+#### Withdrawing Rewards
+
+If you would like to withdraw your rewards, you'll need to contact Kraken's support team.
+
+---
+
+## Finoa (Coming Soon!)
+
```

Comment:
Say something like instructures will be provided as soon as they are ready and will be linked here.